lona/html/parsing.py

```diff
@@ -111,23 +111,22 @@ def get_node_class(self, tag_name, attributes):
         return Node
 
     def handle_starttag(self, tag, attrs):
+        # tag overrides
         node_kwargs = {}
 
-        # node attributes
-        for name, value in attrs:
-            if value is None:
-                value = 'true'
-
-            node_kwargs[name] = value
-
-        # tag overrides
         node_kwargs['tag_name'] = tag
         node_kwargs['self_closing_tag'] = tag in SELF_CLOSING_TAGS
 
         # setup node
         node_class = self.get_node_class(tag, node_kwargs)
         node = node_class(**node_kwargs)
 
+        for name, value in attrs:
+            if value is None:
+                value = 'true'
+
+            node.attributes[name] = value
```

Comment:
This fails for following attributes:
```py
HTML('<div id="abc">')
HTML('<div class="abc">')
HTML('<div style="color: red;">')
```

Reply:
@maratori Ack -.- we need tests for all these cases